types/ssh2-sftp-client/index.d.ts

```diff
@@ -49,4 +55,14 @@ declare namespace sftp {
         owner: number;
         group: number;
     }
+
+    interface FileStats {
```

Comment:
Does than the `FileInfo` extend `FileStats` somehow?

Reply:
Yes, it actally does though FileInfo does not contain the members 'mode' and 'permissions'. Should I add the inheritance then (with mode and permissions as optional members)? And: Is it even possible to add a new commit to an existing pull request?

Reply:
If there's no real inheritance for `FileStats` from `FileInfo` than is ok to have 2 different interfaces.

Of course you can, just keep committing on this branch and push!

Reply:
Fine, than this could be merged imo.